module.js

```diff
@@ -217,19 +211,25 @@ instance.prototype.GetPlaybackState = function(){
             self.checkFeedbacks('is-shuffle');
         }
 
-        let songProgress = data.body.progress_ms;
-        let songDuration = data.body.item.duration_ms
-        let songPercentage = songProgress/songDuration;
-        
-        songPercentage = songPercentage*100;
-        songPercentage = songPercentage.toFixed(0);
-
-        songProgress = songProgress/1000;
-        songProgress = songProgress.toFixed(0);
-
-        songDuration = songDuration/1000;
-        songDuration = songDuration.toFixed(0);
+        if (data.body.item.duration_ms) {
```

Comment:
This should be `if (data.body.item) {` as the problem was it was undefined causing trying to read the `.duration_ms` property to throw an error

Reply:
Fixed this.